src/main/java/TaskManager.java

```diff
@@ -0,0 +1,121 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TaskManager {
+    private final List<Task> recordedTasks;
+    private int taskCount = 0;
```

Comment:
You can consider shfiting taskCount as a static variable in Task class as per what prof mentioned during class